apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/common/AbstractTransformerV2.java

```diff
@@ -106,14 +106,45 @@ protected static List<HumanName> createHumanNameFrom(PatientInfo patientInfo) {
         givens = null;
       }
 
-      names.add(new HumanName().setFamily(patientInfo.getLastName()).setGiven(givens));
+      HumanName name =
```

Comment:
@KyRobbins A small percentage of our gender codes are going to be nonsense values (eg "`").  We need to adjust our mapping logic so when we get a nonsense value we also map to the "unknown" gender code.